pkg/cryptomaterial/certpaths.go

```diff
@@ -0,0 +1,41 @@
+package cryptomaterial
```

Comment:
What's the reasoning behind creating a new go package for this? Would it make sense to add it to the util package?

Reply:
Having a separate package with a specific purpose seemed like a good practice.

I suspect it could exist as a package within the util packge, I don't have a strong opinion on its location.

Reply:
If you're ok with it, let's put it under util where all the helper functions are located.

Reply:
It is discouraged to have a util or commong pkg in Go.

Reply:
I think as long as we keep it structured, we might be ok. Currently it is just one package which does not really play well with the best practices, but I think `util/cryptomaterial` might be ok.